plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/settings/CodeWhispererConfigurable.kt

```diff
@@ -87,6 +92,20 @@ class CodeWhispererConfigurable(private val project: Project) :
                     bindSelected(codeWhispererSettings::isImportAdderEnabled, codeWhispererSettings::toggleImportAdder)
                 }.comment(message("aws.settings.codewhisperer.automatic_import_adder.tooltip"))
             }
+
+            row {
+                link("Configure inline suggestion keybindings") { e ->
+                    val settings = DataManager.getInstance().getDataContext(e.source as ActionLink).getData(Settings.KEY) ?: return@link
+                    val configurable: Configurable = settings.find("preferences.keymap") ?: return@link
+
+                    settings.select(configurable, Q_INLINE_KEYBINDING_SEARCH_TEXT)
+
+                    // workaround for certain cases for sometimes the string is not input there
+                    EdtExecutorService.getScheduledExecutorInstance().schedule({
+                        settings.select(configurable, Q_INLINE_KEYBINDING_SEARCH_TEXT)
+                    }, 500, TimeUnit.MILLISECONDS)
```

Comment:
feels like this should not be required

Reply:
last time I checked if I don't have this, it won't input the text in the keymap setting bar, I will try again later